Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
+        /// The number of the issue.
+        public let number: Int
 
-    /// The ISO8601 date string for when the issue was created.
-    public let createdAt: Date
+        /// The title of the issue.
+        public let title: String
 
-    /// The ISO8601 date string for when the issue was updated.
-    public let updatedAt: Date
+        /// The user who created the issue.
+        public let user: User
 
-    /// The ISO8601 date string for when the issue was closed.
-    public let closedAt: Date?
+        /// The state for the issue: open, closed, locked.
+        public let state: State
 
-    /// The labels associated with this issue.
-    public let labels: [GitHubIssueLabel]
-}
+        /// A boolean indicating if the issue has been locked to contributors only.
+        public let isLocked: Bool
 
-// MARK: - GitHubIssueLabel
+        /// The markdown body message of the issue.
+        public let body: String?
 
-public struct GitHubIssueLabel: Decodable, Equatable {
-    // MARK: - Properties
+        /// The comment number of comments for the issue.
+        public let commentCount: Int
 
-    /// The id number of this label.
-    public let id: Int
+        /// The user who is assigned to the issue.
+        public let assignee: User?
 
-    /// The URL that links to this label.
-    public let url: String
+        /// The users who are assigned to the issue.
+        public let assignees: [User]
 
-    /// The name of the label.
-    public let name: String
+        /// The milestone of this issue
+        public let milestone: Milestone?
 
-    /// The color associated with this label.
-    public let color: String
-}
+        /// The ISO8601 date string for when the issue was created.
+        public let createdAt: Date
 
-// MARK: - GitHubMilestone
+        /// The ISO8601 date string for when the issue was updated.
+        public let updatedAt: Date
 
-public struct GitHubMilestone: Decodable, Equatable {
-    // MARK: - CodingKeys
+        /// The ISO8601 date string for when the issue was closed.
+        public let closedAt: Date?
 
-    enum CodingKeys: String, CodingKey {
-        case closedAt = "closed_at"
-        case closedIssues = "closed_issues"
-        case createdAt = "created_at"
-        case creator
-        case description
-        case dueOn = "due_on"
-        case id
-        case number
-        case openIssues = "open_issues"
-        case state
-        case title
-        case updatedAt = "updated_at"
+        /// The labels associated with this issue.
+        public let labels: [Label]
     }
+}
 
-    // MARK: - MilestoneState
+extension GitHub {
+    public struct Milestone: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case closedAt = "closed_at"
+            case closedIssues = "closed_issues"
+            case createdAt = "created_at"
+            case creator
+            case description
+            case dueOn = "due_on"
+            case id
```

Comment:
<!--
  0 failure: 
  1 warning:  Enum element name...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 411;
-->

- :warning: Enum element name should be between 3 and 40 characters long: 'id' (`identifier_name`)